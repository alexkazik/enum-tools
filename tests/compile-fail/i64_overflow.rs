#![no_implicit_prelude]

use ::enum_tools::EnumTools;

#[derive(EnumTools, Debug, Clone, Copy, Eq, PartialEq)]
#[repr(u64)]
pub(crate) enum EG {
    #[enum_tools(rename = "A*")]
    A = 0x7fff_ffff_ffff_fffe,
    B,
    C,
    D,
}

fn main() {}
