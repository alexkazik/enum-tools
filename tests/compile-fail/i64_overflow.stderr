error: i64 overflow
  --> tests/compile-fail/i64_overflow.rs:11:5
   |
11 |     C,
   |     ^
